libs/coin-modules/coin-evm/src/__tests__/coin-tester/scenarios/ethereum.ts

```diff
@@ -210,15 +213,17 @@ export const scenarioEthereum: Scenario<EvmTransaction> = {
       addressToImpersonnate: "0xa3cd1123f4860C0cC512C775Ab6DB6A3E3d1B1Ee",
     });
 
-    await getLogs();
-
     return {
       currencyBridge,
       accountBridge,
       account: scenarioAccount,
       onSignerConfirmation,
     };
   },
+  getTransactions: address => makeScenarioTransactions({ address }),
+  beforeSync: async () => {
```

Comment:
Just FYI in the polkadot PR there's a `mockIndexer` function with the same purpose

Reply:
:ah: 
Ok then I more in favour of keeping `beforeSync` as a more generic name for this hook 🤷‍♂️